generators/client/templates/angular/src/main/webapp/app/core/language/language.helper.ts.ejs

```diff
@@ -29,6 +30,7 @@ import { FindLanguageFromKeyPipe } from 'app/shared/language/find-language-from-
 @Injectable()
 export class JhiLanguageHelper {
     renderer: Renderer2 = null;
+    private _language: BehaviorSubject<string>;
```

Comment:
@deepu105 is this good to use "_" for local variables?

Reply:
ok since there is a getter for this